### ISSUE TITLE
Genesis plus 1.2a emulator for DOS crashes when enabling sound

### ISSUE THREAD

**Issue by ualaa1** [joncampbell123/dosbox-x#3177]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar bug report(s) already exists?

- [X] I have searched and didn't find any similar issues.

### What operating system(s) this bug have occurred on?

Windows 10 64 bit

### What version(s) of DOSBox-X have this bug?

0.83.20

### Describe the bug

Emulator crashes when enabling sound.  When not enabling sound, it runs fine.
It also generated an error file.
[ERROR.LOG](https://github.com/joncampbell123/dosbox-x/files/7776643/ERROR.LOG)
![command_002](https://user-images.githubusercontent.com/68984176/147401251-0058edff-ff8f-49f5-8fb0-32fc32d727f7.png)

### Expected behavior

Probably for the game to load up with sound

### Steps to reproduce the behaviour

Assuming that the emulator, CWSDPMI and game are in the same folder. Load gens game.bin/md -sound
Watch emulator crash.

**Comment by Wengier** [comment]
I can confirm this, although I do not think the generated error file contains detailed information of the error.

**Comment by grapeli** [comment]
There is sound for me.

https://user-images.githubusercontent.com/452325/147417466-1ad7a729-1244-4119-af83-0eba5a719a53.mp4

Although in sonic it does not sound perfect.
I did not compare what it looks like by running other ports.

**Comment by Wengier** [comment]
@grapeli Right, you need to specify ``-sound on`` instead of ``-sound``, and optionally with a sound card type. Then it will no longer crash.

**Comment by grapeli** [comment]
@Wengier That's right.
Just because a program being launched exits with an error does not necessarily mean that dosbox-x is at fault.

Sonic game audio sounds better by running genesis with a different video output `-vdriver safe` or `modex`.

**Comment by ualaa1** [comment]
Sound works thanks!
And here's proof it does (Don't mind the graphic glitches, the game I'm running pushes the Genesis to its limits and is really hard to emulate): https://imgur.com/a/iFurFco
